queuebacca-sqs/src/main/java/io/axonif/queuebacca/sqs/SqsCourierMessageBinRegistry.java

```diff
@@ -50,20 +51,33 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SqsCourierMessageBinRegistry.class);
 
+    private static final String SQS_TAGGING_RETRY_ATTEMPTS_PROPERTY = "queuebacca.sqs.tagging.retryAttempts";
+    private static final String SQS_TAGGING_RETRY_FREQUENCY_PROPERTY = "queuebacca.sqs.tagging.retryFrequency";
+
+    private static final int SQS_TAGGING_RETRY_ATTEMPTS_DEFAULT = 5;
```

Comment:
I wonder if this is enough?  If we're in a throttle situation, we may want to either delay a bit longer than this, or retry a few more times?

Reply:
Since its limit is 5 transactions per second for tagging, I figured this would be enough tries. However, if we find we need more we can always bump up the amount, that's why I made it configurable, this was just a default.